test/unit/org/apache/cassandra/hints/HintsStoreTest.java

```diff
@@ -137,6 +138,26 @@ public void testConcurrentDeleteExpiredHints() throws Exception {
         assertFalse("All hints files should be deleted", store.hasFiles());
     }
 
+    @Test
+    public void testPendingHintsInfo() throws Exception
+    {
+        HintsStore store = HintsCatalog.load(directory, ImmutableMap.of()).get(hostId);
+        assertEquals(new PendingHintsInfo(store.hostId, 0, Long.MAX_VALUE, Long.MIN_VALUE),
```

Comment:
I'm not sure whether "empty" pending hints infor should be like this or `null`. @ekaterinadimitrova2 please, share your thoughts.

Reply:
I like your idea better! Let's use `null` instead of a dummy object - 6f5f1e22cb23237bb07e76a5e38b20d9965dcf00.